src/js/components/common/dialogs/deviceconnectiondialog.js

```diff
@@ -174,8 +175,9 @@ export const DeviceConnectionDialog = ({ onCancel }) => {
                 Next
               </Button>
             ) : (
-              <Button variant="contained" disabled={!onboardingComplete} onClick={onCancel}>
+              <Button variant="contained" disabled={!onboardingComplete} onClick={onCancel} style={{ height: '37px' }}>
                 {onboardingComplete ? 'Close' : 'Waiting for device'}
+                <Loader show={!onboardingComplete} small style={{ marginTop: '-5px' }} />
```

Comment:
```suggestion
              <Button variant="contained" disabled={!onboardingComplete} onClick={onCancel} endIcon={!onboardingComplete && <Loader show small table style={{ top: -24 }} />}>
                {onboardingComplete ? 'Close' : 'Waiting for device'}
```
with the `24px` resetting the `top: -5px` from the loader container and making room for the `margin-top: 24px` from the loader itself